CONTRIBUTING.md

```diff
@@ -1,102 +1,218 @@
-# Contributing
+# How can I contribute to Candy-doc project ?
 
-Thanks for taking the time to contribute! 😻
+First of all, all the team thank you for taking the time to contribute! 😻
 
-## Fix bugs by creating PR's
+We are really happy you join the community and we hope you'll enjoy working on this project.
 
-We appreciate every time you report a bug in the product. However, taking time to submit a PR can help us in fixing bugs quickly and ensure a healthy and stable eco-system.
+In order to provide the most convenient collaboration, here is a tutorial and some directives to follow to ensure a good organization.
```

Comment:
`[...] to ensure a good organization` may be redundant with the start of the sentence.